data/quickstarts/create-aikit-notebook-quickstart.yaml

```diff
@@ -0,0 +1,63 @@
+kind: ConsoleQuickStart
+metadata:
+  name: create-aikit-notebook
+spec:
+  displayName: Creating an AI Kit notebook
+  durationMinutes: 5
+  icon: 'images/intel.svg'
+  description: This quick start will walk you through creating an AI Kit notebook.
+  introduction: |-
+    ### This quick start shows you how to create an AI Kit notebook.
+    Red Hat® OpenShift® Data Scientist lets you run AI Kit notebooks on our Red Hat® OpenShift Dedicated environment.
+  tasks:
+    - title: Launch AI Kit Notebook
+      description: |-
+        ### To find the AI Kit Launch action:
+        1. In the **Applications** menu item, go to **Enabled** projects.
+        2. Find the AI Kit entry.
+        3. Click **Launch** on the AI Kit card to access the AI Kit Spawner.
+
+        A new browser tab window will open displaying the AI Kit Spawner.
+      review:
+        instructions: |-
+          #### To verify you have launched the AI Kit Spawner:
+          Is a new browser tab visible that reads Spawner Options?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      summary:
+        success: You have launched the AI Kit Spawner!
+        failed: Try the steps again.
+    - title: Spawn an environment
+      description: |-
+        ### Spawning an environment:
+        1. Select a notebook image from the dropdown menu.
+        2. Select a container size based on your computation needs.
+        3. Click the **Start** button.
+
+        The Spawner Options page will reload and indicate that the system is starting up.
+      review:
+        instructions: |-
+          #### To verify that you have launched the AI Kit notebook:
+          Do you see an visual on the Spawner options screen that says the server is running?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      summary:
+        success: You have verified that the AI Kit Spawner has started!
+        failed: Try the steps again.
+    - title: Create your first notebook
+      description: |-
+        ### To create a notebook, follow these steps:
+        1. Under the Notebook display click the Python 3 tile. 
+        2. When the new AI Kit notebook opens, verify that you see Python 3 in the upper right navigation bar.
+
+        You have now launched an AI Kit tnotebook and can begin writing Python.
+      review:
+        instructions: |-
+          #### Verify that your AI Kit notebook launched with a Python 3 kernel:
+          Is Python 3 displaying in the upper right notification bar of your notebook?
+        failedTaskHelp:
+          This task isn’t verified yet. Try the task again.
+      summary:
+        success: You have added health checks to your sample app!
+        failed: Try the steps again.
+  conclusion: You are now able to launch AI Kit notebooks and write Python code.  If you want to learn how to deploy
+    a model, take the next quick start.
+  nextQuickStart: [deploy-python-model]
```

Comment:
I think we want to add AIKit as its own PR.

Reply:
removed